scripts/company.js

```diff
@@ -0,0 +1,117 @@
+const currentUrl = window.location.href;
+const searchParams = new URL(currentUrl).search;
+const searchValue = new URLSearchParams(searchParams);
+const queryString = searchValue.get("symbol");
+const companyImage = document.getElementById("company-image");
+const nameOfCompany = document.getElementById("company-name");
+const companySector = document.getElementById("company-sector");
+const companyIndustry = document.getElementById("company-industry");
+const companyStockPrice = document.getElementById("stock-price");
+const companyDescription = document.getElementById(
+  "company-description-container"
+);
+const companyLink = document.getElementById("company-link");
+const stockChange = document.getElementById("stock-change");
+const changeWraper = document.getElementById("change-wraper");
+const changeSign = document.getElementById("change-sign");
+const chartCanvas = document.getElementById("myChart");
+const loadingSpinner = document.getElementById("loadingSpinner");
+
+let baseUrl = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/`;
+
+function enableSpinner() {
+  loadingSpinner.classList.remove("d-none");
+}
+function disableSpinner() {
+  loadingSpinner.classList.add("d-none");
+}
+
+let history;
+let profile;
+let image;
+async function getCompanyProfile() {
```

Comment:
this function is already written in mainPage.js
use another file for shared code (DRY)